docs/quality-assurance/security-testing-strategy.md

```diff
@@ -0,0 +1,66 @@
+# UKHO Security Testing Strategy
+*Note: This document is the property of the United Kingdom Hydrographic Office and may contain third party material.  It must not be copied or transmitted to any third parties without the written permission of the Owner(s).  :copyright: Crown Copyright 2019*
+
+## Introduction
+The purpose of this document is to outline, at a high level, how Security Testing will be carried out across the UKHO. This will include the tools and technologies used across the various stages of the software development lifecycle.
+
+The Test Community focus will be on Web Application Penetration Testing.
+
+Efforts will not be focused on networking or infrastructure. Security Testing of networking or infrastructure falls into ITSO domain, so we should work with ITSO on this.
+
+> This is a working document and as such, will be updated to stay in line with our evolving Security Testing capability within the test community.
+
+## Current Situation
+There is currently very little formal security testing conducted by members of the Test Community. Testers are involved with Threat Modelling sessions that are starting to be done as a standard across development teams, these however predominantly led by developer security champions. Some testers do then carry out the manual security testing based on Threat Modelling outcomes, working closely with the security champion in the team.
+
+## What is Security Testing?
+Security Testing is a type / area of testing that aims to expose potential vulnerabilities of the system / application under test and determine that its data and resources are protected from potential attacks - to build trust and resilience in applications
+
+Application security ‘testing’ can be split into two distinct areas:
+ - Things we can define/test   
+ - Things we need experts for
+
+This document will focus on the things we can define/test.
+
+## Manual Security Testing
+This will predominantly be focusing on the OWASP Top 10 and based on the outcomes of the development teams Threat Modelling sessions.
+
+OWASP Top 10:
+ 1. Injection 
+ 2. Broken Authentication 
+ 3. Sensitive data exposure
+ 4.  XML External Entities (XXE)
+ 5. Broken Access control 
+ 6. Security misconfigurations
+ 7. Cross Site Scripting (XSS) 
+ 8. Insecure Deserialization 
+ 9. Using Components with known vulnerabilities 
+ 10. Insufficient logging and monitoring
+
+There will also be focus on utilising Security Testing techniques during exploratory testing, where it is applicable – this will grow overtime with knowledge and experience within the Test Community.
+
+## Automated Dynamic Security Testing
```

Comment:
This whole section, is it too detailed. SHould our strategy be at a higher level and more vague. Remembering this is available for the world to see, are we not detailing too much info on how we test so people have an idea where remaining risks may lie..